sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -67,6 +68,10 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
               default: InstitutionDashboard,
               sidebar: InstitutionHomeSideBar,
             },
+            meta: {
+              requiresAuth: true,
```

Comment:
Would make sense to invert the logic here? I mean, consider that everything requires authentication unless that explicit defined as "requiresAuth: false".
I believe that the majority of pages will be under authentication, so we would avoid writing "requiresAuth: true" and if some dev forgets it on a page that is public, it will be easily noted.

Reply:
Actually, for `requiresAuth: false` or if `requiresAuth` is not given, the logic to allow `next()`, if `requiresAuth: true `, logic to check, whether the user is authenticated or not, if authenticated a set of checks else redirecting to `Login` page, let me know if this logic doesn't make sense

Reply:
There is nothing wrong with the logic, my recommendation was to, in case the requiresAuth is not present, consider that by default the authentication is required and just change the logic to execute the authentication check to something like below. In this way, we do not need to explicitly set "requiresAuth: true". That is a recommendation only 😊 

```
if(metadat.requiresAuth !== false)
```